src/test/java/Practice/Selenium_homework3.java

```diff
@@ -0,0 +1,142 @@
+package Practice;
+
+import Helper.Misc;
+import Web.MyDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Selenium_homework3 {
+
+    //Testcase - 1
+    //darksky.net - Verify correct temperature value is displayed after changing temperature units
+
+    @Test
+
+    public static void darkskyTempConversion () {
+
+        MyDriver.launchUrlOnNewWindow("https://darksky.net/");
+
+        //form[@id = 'searchForm']/following-sibling::div[@class = 'options']/div[contains(@class, 'units')]/div[contains(@class, 'hide')]/following-sibling::div[@class = 'selectric']/b[@class = 'button']
+        //div[contains(@class, 'selectric-below')]//b
+        WebElement arrowDown = MyDriver.getDriver().findElement(By.xpath("//form[@id = 'searchForm']/following-sibling::div[@class = 'options']/div[contains(@class, 'units')]/div[contains(@class, 'hide')]/following-sibling::div[@class = 'selectric']/b[@class = 'button']"));
```

Comment:
Good work above to create xpath but try to make it little shorter. You may use this (//b[@class = 'button'])[1]